packages/openapi-generator/scripts/generate.sh

```
@@ -0,0 +1,59 @@
+  [ -f "$file" ] || continue
+  name=$(basename "$file" .json)
+  echo "Generating: $name"
+
+  # Build java args
+  JAVA_ARGS="-cp $GENERATOR_JAR:$CLI_JAR"
+  JAVA_ARGS="$JAVA_ARGS -DgeneratedControllerPath=$OUTPUT_DIR/generated-controller"
+  if [ -n "$CUSTOM_PATH" ]; then
+    JAVA_ARGS="$JAVA_ARGS -DcustomPath=$(realpath -m "$CUSTOM_PATH")"
+  fi
+
+  java $JAVA_ARGS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)